app.py

```diff
@@ -88,5 +104,28 @@
     health_check_interval=5,
 )
 
-# Generate stacks
+app_service_props_otel_collector = ServiceProps(
+    container_name="otel-collector",
+    container_port=4318,
+    container_memory=512,
+    container_location="ghcr.io/sage-bionetworks/sage-otel-collector:v0.0.4",
+    container_env_vars={},
+    container_secrets=[
+        ServiceSecret(
+            secret_name=f"{stack_name_prefix}-DockerFargateStack/{environment}/opentelemetry-collector-configuration",
```

Comment:
the name is currently set to  `schematic-dev-sandbox-DockerFargateStack/dev-sandbox/opentelemetry-collector-configuration` in org-sagebase-dnt-dev account which does not match this format.

Reply:
I was using dev-sandbox for testing. I'll swap the name over for it shortly

Reply:
Name was swapped